src/components/layout/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import {
+  colors,
+  dimensions,
+  respondFrom,
+  respondTo,
+  breakpoints,
+} from "../../styles";
+
+import PageContentContainer from "../ui/PageContentContainer";
+
+import Facebook from "../../assets/svg/Facebook.svg";
+import Instagram from "../../assets/svg/Instagram.svg";
+import Twitter from "../../assets/svg/Twitter.svg";
+
+const FooterMain = styled.footer`
+  width: 100%;
+  background-color: ${colors.white};
+  padding: 20px 0;
+
+  ${respondFrom(breakpoints.desktop)`
+        padding: 33px 0;
```

Comment:
those padding values are quite inconsistent, I would stick to multiplies of 8, so 32px

Reply:
I change it to new spacing variable based on 8 px